src/pages/wonyoungcho/Login/Login.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import './Login.scss';
+
+class Login extends React.Component {
+  goToMain = () => {
+    this.props.history.push('/Mainwyc');
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      idInputValue: '',
+      pwInputValue: '',
+    };
+  }
+
+  loginClick = () => {
+    console.log(this.state.idInputValue);
+    fetch('http://10.58.7.181:8000/user/login', {
+      method: 'POST',
+      headers: {
+        Authorization: localStorage.getItem('access_token'),
+      },
+      body: JSON.stringify({
+        email: this.state.idInputValue,
+        password: this.state.pwInputValue,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        alert(result.message);
+        if (result.message === 'SUCCESS') {
+          this.props.history.push('/Mainwyc');
+        }
+        localStorage.setItem('access-token', result.ACCESS_TOKEN);
+      });
```

Comment:
```suggestion
      .then(result => {
        alert(result.message);
        if (result.message === 'SUCCESS') {
          localStorage.setItem('access-token', result.ACCESS_TOKEN);
          this.props.history.push('/Mainwyc');
        }
      });
```
데이터 통신의 결과가 성공적으로 이루어진 후에 로컬스토리지에 저장하는 과정이 들어가야 하므로 위치를 조정해주면 좋겠네요!